components/automate-cli/pkg/verifyserver/server/api/v1/handler.go

```diff
@@ -25,3 +27,8 @@ func (h *Handler) AddBatchCheckService(bc batchcheckservice.IBatchCheckService)
 	h.BatchCheckService = bc
 	return h
 }
+
+func (h *Handler) AddSoftwareVersionService(sv softwareversionservice.ISoftwareVersionService) *Handler {
+	h.SoftwareVersionService = sv
+	return h
+}
```

Comment:
Use go linter and add a new line at the end of the file.

Reply:
Done